frontend/src/main/components/Driver/DriverAvailabilityForm.js

```diff
@@ -2,131 +2,141 @@ import { Button, Form } from 'react-bootstrap';
+          {errors.day?.message}
+        </Form.Control.Feedback>
+      </Form.Group>
 
-            <Form.Group className="mb-3" >
-                <Form.Label htmlFor="notes">notes</Form.Label>
-                <Form.Control
-                    data-testid={testIdPrefix + "-notes"}
-                    id="notes"
-                    type="text"
-                    isInvalid={Boolean(errors.notes)}
-                    {...register("notes", {
-                        required: "notes is required."
-                    })}
-                />
-                <Form.Control.Feedback type="invalid">
-                    {errors.notes?.message}
-                </Form.Control.Feedback>
-            </Form.Group>
+      <Form.Group className="mb-3">
+        <Form.Label htmlFor="startTime">Start Time</Form.Label>
+        <Form.Control
+          data-testid={testIdPrefix + "-startTime"}
+          id="startTime"
+          type="text"
+          placeholder="Enter time in the format HH:MM AM/PM (eg. 11:30 AM)"
+          isInvalid={Boolean(errors.startTime)}
+          {...register("startTime", {
+            required: "Start Time is required.",
+            pattern: {
+              value: time_regex,
+              message: "Invalid time format. Use HH:MM AM/PM.",
+            },
+          })}
+        />
+        <Form.Control.Feedback type="invalid">
+          {errors.startTime?.message}
+        </Form.Control.Feedback>
+      </Form.Group>
 
-            <Button
-                type="submit"
-                data-testid={testIdPrefix + "-submit"}
-            >
-                {buttonLabel}
-            </Button>
-            <Button
-                variant="Secondary"
-                onClick={() => navigate(-1)}
-                data-testid={testIdPrefix + "-cancel"}
-            >
-                Cancel
-            </Button>
+      <Form.Group className="mb-3">
+        <Form.Label htmlFor="endTime">End Time</Form.Label>
+        <Form.Control
+          data-testid={testIdPrefix + "-endTime"}
+          id="endTime"
+          type="text"
+          placeholder="Enter time in the format HH:MM AM/PM (eg. 5:30 PM)"
+          isInvalid={Boolean(errors.endTime)}
+          {...register("endTime", {
+            required: "End Time is required.",
+            pattern: {
+              value: time_regex,
+              message: "Invalid time format. Use HH:MM AM/PM.",
+            },
+          })}
+        />
+        <Form.Control.Feedback type="invalid">
+          {errors.endTime?.message}
+        </Form.Control.Feedback>
+      </Form.Group>
 
-        </Form>
+      <Form.Group className="mb-3">
+        <Form.Label htmlFor="notes">Notes</Form.Label>
+        <Form.Control
+          data-testid={testIdPrefix + "-notes"}
+          id="notes"
+          type="text"
+          placeholder="Include any additional notes here."
+          isInvalid={Boolean(errors.notes)}
+          {...register("notes", {
```

Comment:
Consider making this optional.  I don't think making this required is going to be a good user experience.